client/components/ResponsiveDrawer.js

```diff
@@ -183,12 +186,6 @@ const ResponsiveDrawer = ({
   );
 };
 
-ResponsiveDrawer.propTypes = {
-  /**
-   * Injected by the documentation to work in an iframe.
-   * You won't need it on your project.
-   */
-  window: PropTypes.func,
-};
+ResponsiveDrawer.propTypes = PropTypes;
```

Comment:
this looks like it will cause console errors

Reply:
same as other propTypes errors. without this i was not able to commit